### ISSUE TITLE
[Accessibility][Focus Order - Settings]: Extra focus is provided on buttons available under 'Configuration'.

### ISSUE THREAD

**Issue by ShellySoni** [microsoft/BotFramework-Composer#2102]
**User Experience:**
Keyboard users will get confuse if extra focus is provided on buttons available under 'Configuration' when user navigate using up/down arrow keys.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Select 'Settings' from left navigation.
5. Navigate on buttons available under 'Configuration' using up/down arrow keys.
6. Observe the issue.

**Actual Result:**
Extra focus is provided on buttons available under 'Configuration' when user navigate using up/down arrow keys. First tab focus is provided on whole control, after this when user navigate using down arrow key then focus is not visible. Then on again pressing down arrow key, focus gets visible to next button.

**Note:**
When user check with screen reader, then it is observed that along with the extra focus, screen reader read two different roles on the same button i.e when focus is on whole button, then its role as button is announced & when user press down arrow key then its role as link is announced. Please ( Refer:Attachment ).

**Expected Result:**
Extra focus should not be provided on buttons available under 'Configuration'. On pressing down arrow key, focus should directly jump to next button.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Attachment:**
[MAS2.4.3_Settings_Screen reader_Incorrect announcement of role.zip](https://github.com/microsoft/BotFramework-Composer/files/4255385/MAS2.4.3_Settings_Screen.reader_Incorrect.announcement.of.role.zip)

**Comment by beyackle** [comment]
Can repro this, and I'll give a go at fixing it.

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.
**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
